netlogo-headless/src/main/mirror/Mirrorables.scala

```diff
@@ -13,34 +13,38 @@ import org.nlogo.core.AgentVariables.implicitTurtleVariables
 object Mirrorables {
 
   case object Patch extends Kind {
-    object Variables extends Enumeration {
+    object PatchVariables extends Enumeration {
```

Comment:
`abstract class C { def x: AnyRef }; object O extends C { object x }` compiles fine in both 2.11.8 and 2.12.0, as does `abstract class Kind { val Variables: Enumeration } ; case object Patch extends Kind { object Variables extends Enumeration }`, so there must be something else/more here...?